debian/changelog

```diff
@@ -0,0 +1,5 @@
+nvdtools (1.0) precise; urgency=low
+
+  * Initial release.
```

Comment:
I would not call this a release... Feels more like `a .deb package has been added` to me.

Reply:
Sure, I will update it.